modules/go-duckdb/1.7.0/MODULE.bazel

```diff
@@ -0,0 +1,13 @@
+module(
+    name = "go-duckdb",
+    version = "1.7.0",
+)
+
+bazel_dep(name = "gazelle", version = "0.36.0")
+bazel_dep(name = "rules_go", version = "0.48.0")
+
+go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+use_repo(
```

Comment:
just a note that this `use_repo` clause is redundant as it's not importing any repos. no need to fix it in this PR but could fix upstream.